packages/rum/test/specs/index.spec.js

```diff
@@ -90,12 +89,12 @@ describe('index', function() {
       throw new Error('ApmBase test error')
     } catch (error) {
       apmBase.captureError(error)
-      expect(apmServer.sendErrors).not.toHaveBeenCalled()
```

Comment:
I think this still applies just needs to be change to `sendEvents`

Reply:
yeah will add it back.